packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/NavigationItem.tsx

```diff
@@ -52,6 +61,18 @@ const Container = styled(NavigationItemBase)<{
                       }
                   }
               `}
+    ${({ $isRounded }) =>
+        $isRounded &&
+        css`
+            border-radius: ${borders.radii.full};
```

Comment:
Use 50% instead:)

Reply:
Why do we have it as a variable then? :D

Reply:
Also, 50% in question 😳
<img width="1048" alt="image" src="https://github.com/user-attachments/assets/59b73bf9-4852-4ccf-982d-7cd76499e06f">

Reply:
Ok that's weird, the full variable is 100px, right? What should it look like?:D

Reply:
It should look like on the video attached to the desc of this PR

Reply:
Ok, I get how it works now, though it's slightly hacky. Please use the full variable then:)